src/vellum/workflows/descriptors/tests/test_utils.py

```diff
@@ -162,3 +163,34 @@ class Outputs(BaseNode.Outputs):
 def test_resolve_value__happy_path(descriptor, expected_value):
     actual_value = resolve_value(descriptor, FixtureState())
     assert actual_value == expected_value
+
+
+def test_resolve_value__for_falsy_values():
```

Comment:
[nit] Let's also move this to `src/vellum/workflows/nodes/bases/tests/test_base_node.py` since I think this is an issue that affects all nodes not just final output nodes and is not utility specific